src/SWP/Bundle/ContentBundle/Processor/EmbeddedImageProcessor.php

```diff
@@ -113,6 +114,20 @@ public function applyByline(ArticleMediaInterface $articleMedia): string
         return $articleMedia->getByLine();
     }
 
+    private function appendImageCopyrightNotice(ArticleMediaInterface $articleMedia, \DOMElement $figCaptionNode): void
+    {
+        $element = new \DOMElement('span');
```

Comment:
`span` tags shouldn't be added if the `copyrightNotice` is `null` because right now it adds unnecessary, empty `<span></span>` tags in `figcaption`: `<figcaption>Review Bombing<span><\/span><span><\/span><\/figcaption>` (See https://github.com/superdesk/web-publisher/pull/1142/files#diff-3e89047c4aa6c805b52110fcfebc7c3eR136)